src/core/auth/session.service.js

```diff
@@ -1,17 +1,24 @@
 'use strict';
 
-angular.module('core.auth').factory('Session', function () {
+angular.module('core.auth').factory('Session', ['$http', function ($http) {
```

Comment:
use arrow

Reply:
Is a angular service, is creating a new context.